### DISCUSSION TITLE
discord.com need phone verify but not support china phone number and google voice phone number ,use  zulip.com maybe better

### DISCUSSION THREAD

**Discussion by gcxfd** [vlang/v#13673] [General]
<img width="910" alt="image" src="https://user-images.githubusercontent.com/69608605/155265685-1ac7e049-3bff-4654-a85b-5f260055d8ae.png">

discord.com need phone verify but not support china phone number and google voice phone number

use  [zulip](https://zulip.com/) maybe better

**Comment by zSnails** [comment]
What does this even have to do with the v language?

**Comment by gcxfd** [comment]
<img width="832" alt="image" src="https://user-images.githubusercontent.com/69608605/155288507-8ebd5e19-ba64-48e9-b549-4a6bcf3ad7e6.png">
https://discord.gg/vlang

**Comment by medvednikov** [comment]
Strange, phone requirement should be off.

**Comment by ghost** [comment]
I heard that the Chinese government ban discord, isn't that the reason?

**Comment by gcxfd** [comment]
Nothing to do with China blocking. China also blocks Google, but with a VPN it can be accessed normally (Google can verify Chinese cell phone numbers normally).

The international area code of China's cell phone is 86, as you can see in the screenshot above discord.com does not support 86 area code (this is a problem with discord.com).

Then I tried to verify with google voice's phone number, but it still didn't work.

I had some problems learning the v-language and posted on github discussions at https://github.com/vlang/v/discussions/13577 and no one replied.

I see a lot of people online on discord.com and I'd like to ask, but I'm frustrated that I can't access it.

I would suggest https://zulip.com/ as a chat tool, it only requires email registration and no mobile verification.

@kahsa
